security/pkg/nodeagent/sds/server.go

```diff
@@ -165,15 +166,23 @@ func (s *Server) initWorkloadSdsService(options *Options) error { //nolint: unpa
 	}
 
 	go func() {
+		log.Info("Start SDS grpc server")
+		numErrorOutputTime1 := 0
+		numErrorOutputTime2 := 0
 		for {
 			// Retry if Serve() fails
-			log.Info("Start SDS grpc server")
 			if err = s.grpcWorkloadServer.Serve(s.grpcWorkloadListener); err != nil {
-				log.Errorf("SDS grpc server for workload proxies failed to start: %v", err)
+				if numErrorOutputTime1 < maxErrorMessageOutputTimes {
```

Comment:
What's the typical reason retry would help? If it's host file mount readiness issue, sleep can help here.

Reply:
Sleeping mechanism is added.